ethcore/builtin/src/lib.rs

```diff
@@ -548,14 +572,39 @@ impl Bn128Pairing {
 	}
 }
 
+impl Implementation for EIP1962 {
+	/// Can fail in many cases, so leave it for an implementation
```

Comment:
No need for a doc-comment here. Also: I'm not sure what "…so leave it for an implementation" means, can you clarify?